src/index.ts

```diff
@@ -1049,6 +1049,16 @@ export interface Configuration {
    */
   purposes?: Purpose[]
 
+  /**
+   * User subscriptions
+   */
+  dataSubjectSubscriptions?: GetSubscriptionsResponse
```

Comment:
This does NOT belong on the configuration. This should be from `getSubscriptions` below.